dashboard/app/models/levels/level_group.rb

```diff
@@ -196,4 +196,17 @@ def self.get_survey_results(script, section)
       end
     end.compact
   end
+
+  # @param {User} current_user - The currently signed in user
+  # @param {User} user - The optional user we're trying to see the attempt of
+  # @param {Level} level - The sublevel we'd like the last attempt for
+  def self.get_sublevel_last_attempt(current_user, user, level)
```

Comment:
I assume it's handled in the controller, but I wanted to call out that it feels weird to have no permission check here - like we'd let any user view any other user's attempt.

Reply:
I had a similar feeling while I was working on this. It's actually handled in the haml :-/  https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/levels/_level_group.haml#L13. I could duplicate that check here, but would mean needing to pass in the script_level